docs/amber_script.md

```diff
@@ -385,10 +385,10 @@ CLEAR {pipeline}
 EXPECT {buffer_name} IDX _x_ {comparator} _value_+
 
 # Checks that |buffer_name| at |x| has values within |tolerance| of |value|
-# when compared with the given |comparator|. The |tolerance| can be specified
-# as 1-4 float values separated by spaces.
-EXPECT {buffer_name} IDX _x_ TOLERANCE \
-    _tolerance_{1,4} {comparator} _value_+
+# The |tolerance| can be specified as 1-4 float values separated by spaces.
```

Comment:
Missing the last half of the sentence that used to be on line 388

Reply:
Also, how do we apply tolerances if the number of tolerance values differ from the data type.

E.g. I give 2 tolerance scalars, but the data is vec4.   Does it repeat, or fill the remaining tolerance components with 0?  I would expect the latter

Reply:
The last half isn't true anymore. I implemented this as VkScript at the moment where EQ is the only comparitor that takes tolerances. So, you don't really specify the comparitor anymore as much as say EQ all the time.

We can do either, it isn't specified for VkScript and, at the moment it's moot as the original implementation of the verify only ever uses the first value (I filed #515 to fix that). I updated the text to say we'd use the default tolerance (instead of 0) which is  0.000001.